app/src/main/java/org/tokend/template/logic/credentials/persistence/CredentialsPersistence.kt

```diff
@@ -1,11 +1,9 @@
 package org.tokend.template.logic.credentials.persistence
 
-import io.reactivex.Maybe
-import io.reactivex.rxkotlin.toMaybe
 import org.tokend.sdk.keyserver.models.WalletInfo
 import org.tokend.template.logic.credentials.SimpleCredentialsProvider
 
-interface CredentialsPersistence: SimpleCredentialsProvider {
+interface CredentialsPersistence : SimpleCredentialsProvider {
```

Comment:
There are no more "simple" and "complicated" credentials, rename the provider to `CredentialsProvider` and rename it's methods too